README.md

```diff
@@ -66,7 +67,9 @@ Jump to competencies for a particular level:
 
 ### Community & Citizenship
 
-- You do things that benefit your peers and community such as providing feedback on product features, filing detailed bugs, and providing constructive comments.
+- You do things to benefit your peers and community such as participating in meetings, including providing constructive feedback on ideas and initiatives.
+- You advocate for yourself and others when something is unclear or you believe there is an alternate approach worth discussion.
+- You provide feedback on product features, file detailed bugs, and voice constructive comments in small- and large-group discussions.
```

Comment:
re: "constructive comments in small- and large-..."; I don't agree that this needs to be in a verbal public forum at Eng I. The way we phrased this at Epic was "You escalate early and often", with the details of how you did left up to you and your manager.

Reply:
Hmm. Large-group discussions I'm on the fence about, but I would have thought small- person discussions would have thought would be an expectation. Interesting. Will ruminate...

Reply:
@kipprice how about _" You escalate early and often with feedback on product features, product bug reports, and constructive criticism."_? If not, would you mind writing what you're envisioning? 😄